pkg/prometheus/operator.go

```diff
@@ -1835,6 +1835,14 @@ func (c *Operator) selectProbes(p *monitoringv1.Prometheus) (map[string]*monitor
 			)
 			continue
 		}
+		pnKey := fmt.Sprintf("probe/%s/%s", probe.GetNamespace(), probe.GetName())
+		if err = store.AddBearerToken(ctx, probe.GetNamespace(), probe.Spec.BearerTokenSecret, pnKey); err != nil {
+			break
+		}
+
+		if err = store.AddBasicAuth(ctx, probe.GetNamespace(), probe.Spec.BasicAuth, pnKey); err != nil {
+			break
+		}
```

Comment:
We still need to call `store.AddTLSConfig()` if the TLS config isn't nil?

Reply:
Yes , you're right.